types/shapediver/appbuilder.ts

```diff
@@ -84,6 +84,38 @@ export interface IAppBuilderWidgetPropsImage extends IAppBuilderWidgetPropsAncho
 	href?: string
 }
 
+/** Properties of an interaction widget. */
+export interface IAppBuilderWidgetPropsInteraction {
+	/** The settings of the interactions. */
+	interactionSettings?: {
+		interactionTypes?: {
+			drag?: boolean,
+			select?: boolean,
+			hover?: boolean,
+		},
+		multiple?: boolean,
+		output?: string | string[],
+		pattern?: string | string[],
+		groupNodes?: boolean,
+		selectedNodes?: {
+			node?: {
+				output: string,
+				pattern?: string,
+				nodeName: string,
+				path: string
+			},
+			nodes?: {
+				output: string,
+				pattern?: string,
+				nodeName: string,
+				path: string
+			}[]
+		}
+	},
+	/** Name of the parameter that should be used. */
+	parameterName?: string

```

Comment:
@MajorMeerkatThe3rd I suggest to rename this property to `parameter` and change it from `string` to `Partial<IAppBuilderParameterRef, "name" | "sessionId">`